core/lib/generators/solidus/install/install_generator.rb

```diff
@@ -159,6 +148,21 @@ def install_payment_method
       end
     end
 
+    def install_auth_plugin
+      # Doesn't need to add solidus_auth_devise if solidus_bolt is being installed as it already has the gem
```

Comment:
Solidus_auth_devise is also present on [solidus_paypal_commerce_platform](https://github.com/solidusio-contrib/solidus_paypal_commerce_platform/blob/master/Gemfile#L15)? Why we didn't have to skip this part for that gem?

Reply:
Now I see that we have included solidus_auth_devise [here](https://github.com/solidusio-contrib/solidus_bolt/blob/master/solidus_bolt.gemspec#L37). Do we really need it?

Reply:
I have checked the [commit](https://github.com/solidusio-contrib/solidus_bolt/commit/043fb7908d74c6340bdfcccab04faffa3667d26d), and it seems a typo.

Reply:
I'm not sure about that but if this is left unchecked then the installer gives the error that the same gem cannot be installed twice for this application

Reply:
We can maybe remove the `solidus_auth_devise` gem from the gemspec and check if everything is working or not.

If everything works we won't need this extra logic here in the installer as well